app/models/migration/data_migration.rb

```diff
@@ -40,5 +40,9 @@ def in_progress?
     def complete?
       completed_at.present?
     end
+
+    def migration_failures_key
```

Comment:
Can we encapsulate this in the `FailureManager` service? I don't think the model should know about failures